.github/workflows/main.yml

```diff
@@ -179,6 +179,7 @@ jobs:
           # Recursive checkout required to build Solidity contracts and bindings.
           submodules: recursive
       - uses: ./.github/actions/rustup
+      - uses: risc0/foundry-toolchain@2fe7e70b520f62368a0e3c464f997df07ede420f
```

Comment:
Why do we need this?

Reply:
I've removed the `ignore` in the rust example of the Ethereum Relay's README. That imports the Ethereum Relay that in turns imports the ethereum contracts, that requires `forge build` when building the contracts.
As Tim pointed out though, since we now have the same example in the docs, I can remove it from there and instead refer to the documentation for that particular example